overview/default.nix

```diff
@@ -237,9 +237,9 @@ let
     }
     // mapAttrs' (
       name: project:
-      nameValuePair "project/${name}/index.html" {
```

Comment:
I really don't like these kind of URLs. They are longer and they contain the `html` bit which is an implementation detail imo and should be hidden from the user.

Reply:
The hiding can be done at the web server side with rewrites. The document structure is independent of that and should be explicit at the file system level. The extra nesting has no meaning in our document structure and thus shouldn't exist.